src/vs/workbench/parts/files/electron-browser/views/explorerDecorationsProvider.ts

```diff
@@ -31,14 +31,20 @@ export class ExplorerDecorationsProvider implements IDecorationsProvider {
 	}
 
 	provideDecorations(resource: URI): IDecorationData {
-		const fileStat = this.model.roots.filter(r => r.resource.toString() === resource.toString()).pop();
+		const fileStat = this.model.findClosest(resource);
 		if (fileStat && fileStat.nonexistentRoot) {
 			return {
 				tooltip: localize('canNotResolve', "Can not resolve workspace folder"),
 				letter: '!',
 				color: listInvalidItemForeground,
 			};
 		}
+		if (fileStat && fileStat.isSymbolicLink) {
+			return {
+				tooltip: localize('symbolicLlink', "Symbolic link"),
+				letter: '\u21f2'
```

Comment:
Maybe there is a better symbol? Something that looks more like this:

![image](https://user-images.githubusercontent.com/900690/36370462-37c9cbac-155f-11e8-9737-cee867ba83c5.png)